tests/unit/test_testing_farm.py

```diff
@@ -20,31 +20,35 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 import pytest
-from flexmock import flexmock
+import flexmock
+import uuid
 from ogr.abstract import CommitStatus
 from packit.config import JobConfig, JobType, JobConfigTriggerType
 from packit.local_project import LocalProject
 
 from packit_service.models import TFTTestRunModel
-from packit_service.service.events import (
-    TestingFarmResultsEvent,
-    TestingFarmResult,
-    TestResult,
-)
-from packit_service.worker.handlers import TestingFarmResultsHandler
+
+# These names are definately not nice, still they help with making classes
+# whose names start with Testing* or Test* to become invisible for pytest,
+# and so stop the test discovery warnings.
+from packit_service.service.events import TestingFarmResultsEvent as TFResultsEvent
```

Comment:
Good catch. You can rename the actual class.

Reply:
I'd rename it to TFT :)

Reply:
I actually like the current name of the class: it's much easier to read and understand than TF-something.

Reply:
Sorry, no, not this time, no more renaming :)

These aliases are local to this test, if you really think renaming these classes is useful then lets do this in some other PR. But as I said above, the current names are perfectly fine from a readability point of view.